packages/stepfunction-event-logger/lib/index.ts

```diff
@@ -0,0 +1,223 @@
+import { Construct, Duration } from "@aws-cdk/core";
```

Comment:
I get this warning from a build of this package (both packages in fact)

```
lib/index.ts:32:1 - warning JSII5: The type "constructs.Construct" is exposed in the public API of this module. Therefore, the module "constructs" must also be defined under "peerDependencies". This will be auto-corrected unless --no-fix-peer-dependencies was specified.
```

It seems like it's just saying you need to add this module as a peer dependency: https://www.npmjs.com/package/constructs?activeTab=versions